spec/models/organisation_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe Organisation do
+  describe '.find_all' do
+    context 'sorting' do
+      before do
+        create(:edition, document_type: 'organisation', title: 'Organisation C')
+        create(:edition, document_type: 'organisation', title: 'Organisation A')
+        create(:edition, document_type: 'organisation', title: 'Organisation B')
+      end
+
+      it 'returns a list of organisations' do
+        organisations = Organisation.find_all
+        expect(organisations).to all be_a(Organisation)
+      end
+
+      it 'returns a list sorted by title' do
+        organisations = Organisation.find_all
+
+        expect(organisations).to match_array([
```

Comment:
`match_array` ignores the order. So `[1,2,3]` would match `[2,3,1]`. Can you think of a better way to ensure that the order is being kept.

Reply:
Implemented sorted_by matcher.